### PR TITLE
IPv6 support

### LINKED ISSUE #33: ipv6 support

### PR DESCRIPTION
**Pull request by stephen**
Closes #33

### CODE REVIEW

**Comment by stephen** [comment]
Need to figure out how to handle multiple connections racing to acquire

File: lib/rtsp.js [hunk 1]
```diff
@@ -32,13 +34,23 @@ function RtspServer(options, external) {
 
 RtspServer.prototype.connectHandler = function(socket) {
 
+  var statusMessages = {
+    453: "NOT ENOUGH BANDWIDTH"
+  };
+
+  if (this.handling) {
+    socket.end();
+    return;
```
Comment (stephen) [inline lib/rtsp.js]: bleh

**Comment by LinusU** [comment]
Do you need help with anything specific?

**Comment by stephen** [comment]
I think the only thing left before this can land is figuring out what to do when two requests are attempting to connect, but neither has entered the "client connected" stage (i.e. `ANNOUNCE` has been written). It seems sometimes OS X at least sends two requests - one via IPv6 and one via IPv4. 

This needs to reject -only one-, and only send a `453` if there actually is a client already connected. Might need to base it off of requester IP, or similar.

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "start": "node ./examples/server.js",
-    "test": "gulp test"
+    "test": "./node_modules/.bin/gulp test"
```
Comment (LinusU) [inline package.json]: This change shouldn't be needed? `npm` places `node_modules/.bin` in `$PATH` when running `scripts`.

File: package.json [hunk 3]
```diff
@@ -23,8 +23,8 @@
   "dependencies": {
     "debug": "^2.1.0",
     "forge": "^2.2.0",
-    "httplike": "1.0.1",
-    "ip": "0.3.2",
+    "httplike": "1.0.2",
```
Comment (LinusU) [inline package.json]: Any specific reason for not using `^`?

**Comment by LinusU** [comment]
Using requester IP makes sense, I wonder how the AirPort express is coded to handle this?

File: package.json (hunk 2 above)
Comment (stephen) [inline package.json]: Neat! This is indeed unnecessary.

File: package.json (hunk 3 above)
Comment (stephen) [inline package.json]: :+1:

**Pull request merged**